faunadb-java-dsl/src/main/java/com/faunadb/client/query/Language.java

```diff
@@ -354,1261 +484,2003 @@ public static Expr Value(HighPrecisionTime value) {
+   * @see #Value(long)
    */
   public static Expr Epoch(Expr num, Expr unit) {
     return Fn.apply("epoch", num, "unit", unit);
   }
 
   /**
-   * Creates a new Date expression.
+   * Creates a date from an ISO-8601 formatted date string.
    *
+   * @param str an ISO-8601 formatted date string
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#time_functions">FaunaDB Time and Date Functions</a>
+   * @see #Value(String)
    */
   public static Expr Date(Expr str) {
     return Fn.apply("date", str);
   }
 
   /**
-   * Creates a new NextId expression.
+   * Returns a new string identifier suitable for use when constructing references.
    *
    * @deprecated Use NewId() instead.
    *
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#misc_functions">FaunaDB Miscellaneous Functions</a>
+   * @see #Ref(Expr, Expr)
    */
   @Deprecated
   public static Expr NextId() {
     return Fn.apply("next_id", NullV.NULL);
   }
 
   /**
-   * Creates a new NewId expression.
+   * Returns a new string identifier suitable for use when constructing references.
    *
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#misc_functions">FaunaDB Miscellaneous Functions</a>
+   * @see #Ref(Expr, Expr)
    */
   public static Expr NewId() {
     return Fn.apply("new_id", NullV.NULL);
   }
 
   /**
-   * Creates a new Class expression.
+   * Creates a new reference for the given class name.
    *
+   * @param name the class name. Type: String
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#misc_functions">FaunaDB Miscellaneous Functions</a>
    */
   public static Expr Class(Expr name) {
     return Fn.apply("class", name);
   }
 
   /**
-   * Creates a new Class expression.
+   * Creates a new reference for the given class name.
    *
+   * @param name the class name
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#misc_functions">FaunaDB Miscellaneous Functions</a>
    */
   public static Expr Class(String name) {
     return Class(Value(name));
   }
 
   /**
-   * Creates a new Class expression.
+   * Creates a reference for the given class name, scoped to the database provided.
    *
+   * @param name the class name. Type: String
+   * @param database the scope database. Type: Reference
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#misc_functions">FaunaDB Miscellaneous Functions</a>
+   * @see #Database(String)
    */
   public static Expr Class(Expr name, Expr database) {
     return Fn.apply("class", name, "scope", database);
   }
 
   /**
-   * Creates a new Class expression.
+   * Creates a reference for the given class name, scoped to the database provided.
    *
+   * @param name the class name
```

Comment:
Type:  String

Reply:
The name parameter is typed as `String`. The Javadoc will already state its type in the generated html file. I'm only documenting parameters that are typed as `Expr` because those are the ones that Javadoc can't help you with.